prof/src/com/oracle/souffleprof/Tui.java

```diff
@@ -49,8 +49,10 @@ public void runCommand(String[] c) {
         } else if (c[0].equals("rel")) {
             if (c.length == 2) {
                 relRul(c[1]);
-            } else {
+            } else if (c.length == 1) {
                 rel(c[0]);
+            } else {
+                System.out.println("Invalid parameters to rel command");
```

Comment:
This could be a little more user-friendly by e.g. printing the usage for the relevant command as part of the error message (or even just the full help)